src/OpenTK.Graphics/OpenGL/Compatibility/GL.Manual.cs

```diff
@@ -5,6 +5,13 @@ namespace OpenTK.Graphics.OpenGL.Compatibility
 {
     public static unsafe partial class GL
     {
+        // FIXME: Remove this when it's fixed
+        // This is here because there in the gl.xml
+        // one of the paramteres for "glSampleMaskIndexedNV"
+        // is marked with a group named this, but this group is never referenced
+        // anywhere else in the file.
+        public enum SampleMaskNV
```

Comment:
Doesn't the bindings generator have the ability to manually fit overrides like these in? Or does that not work with enums.

Reply:
Read the comments, this one is broken in the .xml files. We will fix it but this fix will do for the first pre-release

Reply:
The only change i did here is making the API's mirror each other, as previously this was only implemented for `Graphics.OpenGL`